applications/maplab-server-node/include/maplab-server-node/maplab-server-node.h

```diff
@@ -54,11 +56,13 @@ class MaplabServerNode final {
   ~MaplabServerNode();
 
   // Once the node is started, the configuration cannot be changed anymore.
-  void start();
+  void start(const bool& load_previous_state);
```

Comment:
built-in types should rather be passed by const value rather than reference. Here, a `const bool&` can be significantly slower than a `const bool` (depending on the c++ implementation).